Extensions/LinkedObjects/Extension.cpp

```diff
@@ -76,12 +76,28 @@ void DeclareLinkedObjectsExtension(gd::PlatformExtension& extension) {
       .SetIncludeFile("LinkedObjects/LinkedObjectsTools.h");
 
   extension
-      .AddCondition("PickObjectsLinkedTo",
+      .AddCondition("QuickPickObjectsLinkedTo",
                     _("Take into account linked objects"),
                     _("Take some objects linked to the object into account for "
                       "next conditions and actions.\nThe condition will return "
                       "false if no object was taken into account."),
-                    _("Take into account all \"_PARAM1_\" linked to _PARAM2_"),
+                    _("Take into account all \"_PARAM2_\" linked to _PARAM3_"),
+                    _("Linked objects"),
+                    "CppPlatform/Extensions/LinkedObjectsicon24.png",
+                    "CppPlatform/Extensions/LinkedObjectsicon16.png")
+
+      .AddCodeOnlyParameter("currentScene", "")
+      .AddCodeOnlyParameter("eventsFunctionContext", "")
```

Comment:
Thinking about this again => Can we just add `.AddCodeOnlyParameter("eventsFunctionContext", "")` at the end of the parameters of the existing `PickObjectsLinkedTo`? Because:

- This is a code only parameter, so it's automatically filled (even if it was missing before).
- It's a new parameter, so it won't disturb the existing ones.

As the implementation is almost exactly the same, it would make sense to keep a single action/condition :)

Reply:
Oh, ok, I didn't know it could be at the end, sounds like a good idea.

Reply:
Yeah it works at any place. Not the prettiest design, but well, that's no big deal as it's a "code only" parameter.